jar-infer/jar-infer-lib/src/test/java/com/uber/nullaway/jarinfer/JarInferTest.java

```diff
@@ -53,7 +53,6 @@
 
 /** Unit tests for {@link com.uber.nullaway.jarinfer}. */
 @RunWith(JUnit4.class)
-@SuppressWarnings("CheckTestExtendsBaseClass")
 public class JarInferTest {
```

Comment:
In this class, can we create unit tests for a method returning an `Object[]` and also for a method in a generic class?  Like `String foo(Object[] o) { return o.toString(); }` for arrays and `String foo(T t) { return t.toString(); }` for generics (where `T` can be a type variable for the enclosing class).  I think the output format for such methods may have changed and we should have tests for those cases.